src/webgl/shaders/point.frag

```diff
@@ -1,7 +1,7 @@
 precision mediump float;
 precision mediump int;
-uniform vec4 color;
-varying float v_strokeWeight;
+uniform vec4 uColor;
```

Comment:
Could this be uMaterialColor to better match other shaders? Or is it a distinctly different property?